test/controllers/declaration/DocumentsProducedControllerSpec.scala

```diff
@@ -187,21 +187,6 @@ class DocumentsProducedControllerSpec
 
       }
 
-      "try to remove a non existent document" in {
```

Comment:
here the same and probably in more cases, but I'll just left comments here :)

Reply:
Similar to my response above

There's two scenarios:
- Remove something that exists
- Remove something that doesnt exist

The result of both is that the thing no longer exists. Does it matter if it ever existed? I don't think so